featureless-void.cabal

```diff
@@ -113,25 +113,7 @@ executable         featureless-void
     hs-source-dirs:    src/app
     build-depends:     base, featureless-void
 
-    ghc-options:       -threaded -rtsopts -with-rtsopts=-N
-
-    extensions: TemplateHaskell
-                QuasiQuotes
-                OverloadedStrings
-                NoImplicitPrelude
-                MultiParamTypeClasses
-                TypeFamilies
-                GADTs
-                GeneralizedNewtypeDeriving
-                FlexibleContexts
-                FlexibleInstances
-                EmptyDataDecls
-                NoMonomorphismRestriction
-                DeriveDataTypeable
-                ViewPatterns
-                TupleSections
-                RecordWildCards
-                CPP
```

Comment:
They really are! All of them are defined on the lib already though, so these are redundant.